index.html

```diff
@@ -18,7 +18,11 @@
   <script>
     (function(window, videojs) {
       var player = window.player = videojs('videojs-vr-player');
-      player.vr({projection: '360', debug: true});
+      player.mediainfo = player.mediainfo || {};
```

Comment:
switched the demo to mediainfo as that is probably the best way to do it (unless you only have one video)